tests/envs/test_registration.py

```diff
@@ -188,112 +193,108 @@ def test_spec_with_kwargs():
+        spec("Unknown1-v1")
     except error.UnregisteredEnv:
         pass
     else:
         assert False
 
 
 def test_malformed_lookup():
-    registry = registration.EnvRegistry()
     try:
-        registry.spec("“Breakout-v0”")
+        spec("“Breakout-v0”")
     except error.Error as e:
         assert "Malformed environment ID" in f"{e}", f"Unexpected message: {e}"
     else:
         assert False
 
 
 def test_versioned_lookups():
-    registry = registration.EnvRegistry()
-    registry.register("test/Test-v5")
+    register("test/Test2-v5")
 
     with pytest.raises(error.VersionNotFound):
-        registry.spec("test/Test-v9")
+        spec("test/Test2-v9")
 
     with pytest.raises(error.DeprecatedEnv):
-        registry.spec("test/Test-v4")
+        spec("test/Test2-v4")
 
-    assert registry.spec("test/Test-v5")
+    assert spec("test/Test2-v5")
 
 
 def test_default_lookups():
-    registry = registration.EnvRegistry()
-    registry.register("test/Test")
+    register("test/Test3")
 
     with pytest.raises(error.DeprecatedEnv):
-        registry.spec("test/Test-v0")
+        spec("test/Test3-v0")
 
     # Lookup default
-    registry.spec("test/Test")
-
-
-def test_env_spec_tree():
-    spec_tree = EnvSpecTree()
-
-    # Add with namespace
-    spec = EnvSpec("test/Test-v0")
-    spec_tree["test/Test-v0"] = spec
-    assert spec_tree.tree.keys() == {"test"}
-    assert spec_tree.tree["test"].keys() == {"Test"}
-    assert spec_tree.tree["test"]["Test"].keys() == {0}
-    assert spec_tree.tree["test"]["Test"][0] == spec
-    assert spec_tree["test/Test-v0"] == spec
-
-    # Add without namespace
-    spec = EnvSpec("Test-v0")
-    spec_tree["Test-v0"] = spec
-    assert spec_tree.tree.keys() == {"test", None}
-    assert spec_tree.tree[None].keys() == {"Test"}
-    assert spec_tree.tree[None]["Test"].keys() == {0}
-    assert spec_tree.tree[None]["Test"][0] == spec
-
-    # Delete last version deletes entire subtree
-    del spec_tree["test/Test-v0"]
-    assert spec_tree.tree.keys() == {None}
-
-    # Append second version for same name
-    spec_tree["Test-v1"] = EnvSpec("Test-v1")
-    assert spec_tree.tree.keys() == {None}
-    assert spec_tree.tree[None].keys() == {"Test"}
-    assert spec_tree.tree[None]["Test"].keys() == {0, 1}
-
-    # Deleting one version leaves other
-    del spec_tree["Test-v0"]
-    assert spec_tree.tree.keys() == {None}
-    assert spec_tree.tree[None].keys() == {"Test"}
-    assert spec_tree.tree[None]["Test"].keys() == {1}
-
-    # Add without version
-    myenv = "MyAwesomeEnv"
-    spec = EnvSpec(myenv)
-    spec_tree[myenv] = spec
-    assert spec_tree.tree.keys() == {None}
-    assert myenv in spec_tree.tree[None].keys()
-    assert spec_tree.tree[None][myenv].keys() == {None}
-    assert spec_tree.tree[None][myenv][None] == spec
-    assert spec_tree.__repr__() == "├──Test: [ v1 ]\n" + f"└──{myenv}: [  ]\n"
+    spec("test/Test3")
+
+
+# def test_env_spec_tree():
```

Comment:
Since `EnvSpecTree` class is deleted, it's probably best to delete this test code as well, instead of commenting out.